lib/crowbar/client/command/role.rb

```diff
@@ -65,16 +80,19 @@ module Role
                 Request.instance.role_show(barclamp, role) do |request|
                   case request.code
                   when 200
-                    body = begin
-                      JSON.parse(request.body)
-                    rescue
-                      []
-                    end
+                    formatter = Formatter::Array.new(
```

Comment:
Uhm, formatter got always different parameters like opts from the current context, specific headings and request from the current context.